shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/metadata/ShardingTableMetaDataBuilder.java

```diff
@@ -173,7 +134,7 @@
     
     private Optional<TableMetaData> loadTableByDataNode(final DataNode dataNode, final DatabaseType databaseType, final Map<String, DataSource> dataSourceMap) {
         try {
-            return TableMetaDataLoader.load(dataSourceMap.get(dataNode.getDataSourceName()), dataNode.getTableName(), databaseType);
+            return DefaultTableMetaDataLoader.load(dataSourceMap.get(dataNode.getDataSourceName()), dataNode.getTableName(), databaseType);
```

Comment:
@tuichenchuxin Is it possible to use TableMetaDataLoaderEngine to load?

Reply:
> @tuichenchuxin Is it possible to use TableMetaDataLoaderEngine to load?

At present, there are still some scenarios where `DefaultTableMetaDataLoader.load` is called separately, which has a lot of impact and has not been modified for the time being. In the future, the logic needs to be unified into the `TableMetaDataLoaderEngine`.